### ISSUE TITLE
Windows version first release

### ISSUE THREAD

**Issue by shoyrock** [FlorianEagox/weeablind#22]
Hi, are there any instructions on how to go about getting the Windows version working? I am unsure if I’m doing something wrong; it’s saying that FFmpeg isn’t detected. Also, my GPU was not detected.

**Comment by FlorianEagox** [comment]
Hi there! First of all, may I know if you're running the release version (the exe) or building it from source with Python? If you don't have ffmpeg installed on your system and in your system path, it will prompt you to download it and add it to the path automatically. Also, your GPU will only be shown as detected if you have the Coqui TTS module installed (i.e. `pip install tts` ) as well as have CUDA set up properly on your system. You can read the SO thread in the readme to set that up.

**Comment by shoyrock** [comment]
Thanks for the reply. I am using the exe version, and yes, it did prompt me to install FFmpeg, but it's still saying that it's not installed even after I closed and reopened the application; I should have CUDA already set. I use it for some llm, but I need to install the TTS module

**Comment by FlorianEagox** [comment]
Oh wow, okay I just tested it, and it worked fine. When it finishes downloading ffmpeg, do you have an ffmpeg folder in the output folder and a "install.crumb" file or something like that? If it finds that, it shouldn't prompt you.

**Comment by shoyrock** [comment]
Yes, FFmpeg and install. Crumb was in the output. it wasn’t giving me a prompt. It was just saying that it couldn’t be found

**Comment by shoyrock** [comment]
Yes, they are bought in the folder, but it’s saying that FFmpeg could not be found or avconv

**Comment by FlorianEagox** [comment]
As in in the terminal like you're getting output errors? Usually I think pydub will run a check and say that because it gets included before the code that runs to add fmpeg to the path, but even when it does that, it still works if it's installed in the output folder.

**Comment by shoyrock** [comment]
If there is a script that I can use to install all the requirements in one go, I keep running into errors

**Comment by FlorianEagox** [comment]
I will try to rebuild the requirements windows to be more robust and test it on another machine. The thing is, there are a few projects that run into conflicts that sometimes you'll get errors about that don't actually matter like protobuf or librosa but pip still acts like a nonce about it. I need to learn a better way to manage that with like conda or something? Wish I knew someone who knew the best way for managing lots of dependencies like that. Generally though, you should be able to install all the features you want individually (i.e. coqui, spleeter, pyannote, etc) without issue. And when something breaks usually it's because of like one package you can just forcefully install or install with --no-deps or something. I'll look into it again. So sorry for the persistence of this issue

**Comment by shoyrock** [comment]
That's one of the problems. That has happened when I try to install the requirements of different parts that need different versions of the same program. I don't have the skill required to fix the problem, so I use an LLM sometimes but then run into other problems

**Comment by FlorianEagox** [comment]
@ArtyomVancyan created issue #3 and might have some insight into fixing these problems? I'm going to play with it right now though and see if I can get something better

**Comment by shoyrock** [comment]
Any assistance would be much appreciated.

**Comment by FlorianEagox** [comment]
Lol trying to reinstall everything to test and all my drives are completely full xD might be a couple days. Need to reinstall my OSes anywways

**Comment by FlorianEagox** [comment]
Ok latest commit might fix some issues. It seems there's a new fork of Coqui being maintained more actively. Installing straight from requirements.txt just worked on 3.10. If that doesn't work, try installing requirements-win

**Comment by shoyrock** [comment]
> Ok latest commit might fix some issues. It seems there's a new fork of Coqui being maintained more actively. Installing straight from requirements.txt just worked on 3.10. If that doesn't work, try installing requirements-win

I’ll give it another try later today and see if it works, I also came across this don’t know if you know about it already https://github.com/2noise/ChatTTS
